cmd/syncthing/main.go

```diff
@@ -1079,7 +1090,8 @@ func standbyMonitor() {
 			// things a moment to stabilize.
 			time.Sleep(restartDelay)
 
-			restart()
+			l.Infoln("Restarting")
+			stop <- exitRestarting
```

Comment:
Again, not related and can be ignored but I think we should init a global controller in init() {} or var, and then use that everywhere oppose to relying on some number of global channels in the main package and some magical struct in other cases.

Reply:
Something like this? https://github.com/syncthing/syncthing/pull/5532